plugins/rikki/heroeslounge/assets/css/heroeslounge.css

```diff
@@ -401,6 +401,10 @@
 .match.format-mini>.record>.opponent>.result.loss {
     color: #e61a1a
 }
+.spoiler {
+    color: gray !important;
```

Comment:
Is there a specific reason for this to need `!important`?

Reply:
the colors for the same elements are the ones just above that have much more specific selectors, which means that without !important the spoiler color would not be used.

It might not be required for the background color, but I figured I'd stick with it just in case.